packages/botonic-nlp/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "@botonic/nlp",
   "version": "0.19.0-alpha.9",
+  "main": "lib/index.js",
```

Comment:
add also 
`"types": "lib/index.d.ts"`
and the following (adapting to your case)
```
"files": [
    "bin/**",
    "lib/**",
    "src/**",
    "doc/**",
    "README.md"
  ],
```